src/s1/components/banner/bannerSlim.jsx

```diff
@@ -0,0 +1,223 @@
+
+		if (window && shouldAutoResizeModal) {
+			window.removeEventListener('resize', this.handleResize);
+		}
+
+		if (this.scrollRef) {
+			this.scrollRef.removeEventListener('scroll', this.handleScroll);
+		}
+
+		debounce.clear();
+	}
+
+	handleAcceptAll = () => {
+		const { store } = this.props;
+		const {
+			tcModel: { consentScreen },
+		} = store;
+
+		logger(LOG_EVENTS.CMPClick, {
+			action: 'click',
+			category: 'acceptAll',
+			label: `screen${consentScreen}`,
+		});
+
+		store.toggleAll();
+	};
+
+	handleLearnMore = () => {
+		const { store } = this.props;
+		const {
+			tcModel: { consentScreen },
+		} = store;
+
+		logger(LOG_EVENTS.CMPClick, {
+			action: 'click',
+			category: 'learnMore',
+			label: `screen${consentScreen}`,
+		});
+
+		store.toggleConsentScreen(CONSENT_SCREENS.STACKS_LAYER1);
+	};
+
+	handleSave = () => {
+		const { store } = this.props;
+
+		store.save();
+
+		const {
+			tcModel: { consentScreen },
+		} = store;
+
+		logger(LOG_EVENTS.CMPClick, {
+			action: 'click',
+			category: 'save',
+			label: `screen${consentScreen}`,
+		});
+	};
+
+	handleResize = debounce(() => {
+		const { store } = this.props;
+		const { maxHeightModal, shouldAutoResizeModal } = store;
+
+		let newMaxHeightModal = maxHeightModal;
+
+		if (shouldAutoResizeModal && this.aboveFoldRef && this.aboveFoldRef.clientHeight) {
+			newMaxHeightModal = this.aboveFoldRef.clientHeight + 100;
+		}
+
+		store.toggleAutoResizeModal(shouldAutoResizeModal, newMaxHeightModal);
+	}, 100);
+
+	handleScroll = debounce(() => {
+		this.setState({
+			hasScrolled: true,
+		});
+
+		if (this.scrollRef) {
+			this.scrollRef.removeEventListener('scroll', this.handleScroll);
+		}
+	});
+
+	render(props, state) {
+		const { hasScrolled } = state;
+		const { isShowing, store } = props;
+		const {
+			config: { theme, showCloseX, },
+			translations,
+			maxHeightModal,			
+		} = store;
+
+		const {
+			isBannerModal,
+			isBannerInline,
+			maxWidthModal,
+			// maxHeightModal, // handled in store
+			primaryColor,
+			primaryTextColor,
+			backgroundColor,
+			textColor,
+			textLightColor,
+			showDropShadow,
+		} = theme;
+
+		const bannerClasses = [style.banner, style.bannerSlim];
+		if (!isShowing) {
+			bannerClasses.push(style.hidden);
+		}
+		if( showDropShadow ) {
+			bannerClasses.push(style.bannerShadow);
+		}
+		if (isBannerModal) {
+			bannerClasses.push(style.bannerModal);
+		} else if (isBannerInline) {
+			bannerClasses.push(style.bannerInline);
+		}
+
+		return (
+			<div
+				class={bannerClasses.join(' ')}
+				style={{
+					backgroundColor,
+					color: textLightColor,
+					...(maxWidthModal ? { maxWidth: maxWidthModal } : {}),
+				}}
+			>
+				<div
+					class={[style.content, style.layer1, hasScrolled ? style.scrolling : ''].join(' ')}
+					ref={(el) => (this.scrollRef = el)}
+					style={{
+						maxHeight: maxHeightModal,
+					}}
+				>
+					{ showCloseX && <div class={style.closeX} onClick={this.handleAcceptAll}>&times;</div>}
```

Comment:
checking with product on this, think compliance didn't want close button to accept all originally but that mighta changed